kubernetes/README.md

```diff
@@ -0,0 +1,86 @@
+# Deploying OpenWhisk on Kubernetes
+
+[![Build Status](https://travis-ci.org/openwhisk/openwhisk-devtools.svg?branch=master)](https://travis-ci.org/openwhisk/openwhisk-devtools)
+
+The collection of scripts and Kubernets pods can be used to
+run OpenWhisk in a Kubernets cluster.
+
```

Comment:
We should add some more basic requirements here and maybe a script or point to the Ubuntu setup script in the openwhisk repo. 

What are the minimum basic openwhisk requirements?
Ansible version?
Any Ansible plugins? docker-py?
Docker version? Can I use docker for Mac?
Ubuntu and Mac only? No windows using vagrant file
Minimum Memory?